src/clr-addons/location-bar/location-bar-node/location-bar-node.component.scss

```diff
@@ -13,7 +13,7 @@
 
     .unselectable-node {
       display: inline-block;
-      line-height: 0.35rem;
+      line-height: 0.45rem;
```

Comment:
pls change it to "line-height: 1rem" and "verticle-align:middle", looks a bit better on all zoom levels